test/WebJobs.Script.Tests.Integration/Host/StandbyManagerTests.cs

```diff
@@ -58,48 +59,48 @@ public async Task StandbyMode_EndToEnd()
                 { EnvironmentSettingNames.AzureWebsiteSku, "Dynamic" },
                 { EnvironmentSettingNames.AzureWebsiteHomePath, null },
                 { EnvironmentSettingNames.AzureWebsiteInstanceId, "87654639876900123453445678890144" },
-                { "AzureWebEncryptionKey", "0F75CA46E7EBDD39E4CA6B074D1F9A5972B849A55F91A248" }
+                { "AzureWebEncryptionKey", "0F75CA46E7EBDD39E4CA6B074D1F9A5972B849A55F91A248" },
+                { EnvironmentSettingNames.AzureWebsiteInstanceId, "87654639876900123453445678890144" }
```

Comment:
Looks like you've added a dupe here? What change are you intending to make?

Reply:
That was an accident. I've fixed it locally.

Reply:
Does anyone know what AzureWebEncryptionKey is about? Isn't it a security concern to have it in the repository.